src/mono/wasm/wasm.proj

```diff
@@ -35,9 +36,8 @@
       />
   </Target>
 
-  <Target Name="BuildWasmRuntimes" DependsOnTargets="BuildPInvokeTable">
+  <Target Name="BuildWasmRuntimes" DependsOnTargets="BuildPInvokeTable" AfterTargets="Build">
```

Comment:
I would just make this target the default target instead of making it `AfterTargets="Build"`